docs/QuickStart.md

```diff
@@ -0,0 +1,58 @@
+## Getting Started / Quick Start
+
+### Features
+
+* Parses Mustache templates and an extended template format (in YAML)
+* Generates a view schema from parsed template data
+* Supports Mustache partials and sections
+* Renders templates with user-provided views
+* Validates user-provided views against generated view schema
+* Includes a [command line interface](#cli)
+
+### CLI
+
+A command line interface is provided via a `fast` binary.
+The help text is provided below and also accessed via `fast --help`:
+
+
+```
+fast <command>
+
+Commands:
+  fast validate <file>                                validate given template source file
+  fast schema <file>                                  get template parameter schema for given template source file
+  fast guiSchema <file>                               get template parameter schema (modified for use with JSON Editor) for given template source file
+  fast validateParameters <tmplFile> <parameterFile>  validate supplied template parameters with given template
+  fast render <tmplFile> [parameterFile]              render given template file with supplied parameters
+  fast validateTemplateSet <templateSetPath>          validate supplied template set
+  fast htmlpreview <tmplFile> [parameterFile]         generate a static HTML file with a preview editor to standard out
+  fast packageTemplateSet <templateSetPath> [dst]     build a package for a given template set
+
+Options:
+  --help     Show help                                                                                         [boolean]
+  --version  Show version number                                                                               [boolean]
+```
+
+For more information on a given command use the `--help` flag combined with a command:
+
+```bash
+fast <command> --help
+```
+
+The CLI can also be accessed by executing `cli.js`.
+For example:
+
+```bash
+./cli.js render path/to/template
+```
+
+### ContentTypes
+
+When BIG-IP FAST renders it is doing string replacement via Mustache, which is agnostic to the output type. However, specifying a contentType in the template can enable some additional features:
+
+* Post-processing steps (e.g., strip dangling commas for JSON)
+* Smarter merges
+* Smarter handling of some data types
+
+#### Note:
+Since BIG-IP FAST relies on BIG-IP AS3 for templating, a contentType of application/json should be declared indicating that the body format is JSON. See [What is the correct JSON Content Type](https://stackoverflow.com/questions/477816/what-is-the-correct-json-content-type) for additional information.
```

Comment:
This note seems more appropriate for Cloud Docs rather than this set of docs.

Reply:
I have removed this note.